### ISSUE TITLE
Material icons with underscore in name aren't displayed in [Meteor] Android App

### ISSUE THREAD

**Issue by fabyeah** [Dogfalo/materialize#1626]
They work in the browser, but not in the [Meteor] Android App.

Can somebody confirm or deny?

Android 4.4.4

**Comment by Alacon** [comment]
I have the same issue.
However none of my icons are visible.

Problems with Google Font?

**Comment by fabyeah** [comment]
Did you define an app access rule in mobile-config.js?

either wildcard
```
App.accessRule('*');
```

or these specifically for only google fonts, i think
```
App.accessRule('https://ajax.googleapis.com'); // google webfonts css
App.accessRule('http://fonts.gstatic.com'); // google webfonts font files
```

**Comment by vhuerta** [comment]
Same problem, only with icons than use underscore in his name

**Comment by Dogfalo** [comment]
Fixed in 0b876f8

**Comment by anirudhaanu** [comment]
use - (hyphen) instead of _ (underscore).
